org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalRequestor.java

```diff
@@ -323,6 +324,11 @@ private void initializeCompletionListItemDefaults(CompletionProposal proposal) {
 		if (completionItem.getTextEdit() != null && preferenceManager.getClientPreferences().isCompletionListItemDefaultsEditRangeSupport()) {
 			itemDefaults.setEditRange(getEditRange(completionItem, preferenceManager));
 		}
+		if (preferenceManager.getClientPreferences().isCompletionListItemDefaultsInsertTextModeSupport() &&
+			preferenceManager.getClientPreferences().getCompletionItemInsertTextModeDefault() != InsertTextMode.AdjustIndentation
+		) {
+			itemDefaults.setInsertTextMode(InsertTextMode.AdjustIndentation);
```

Comment:
This class declares a `CompletionItemDefaults` called `itemDefaults` as well as its member. Is the local variable here on purpose?